juniper_subscriptions/src/lib.rs

```diff
@@ -135,6 +136,108 @@ where
     }
 }
 
+/// Enum of Subscription Protocol Message Types
```

Comment:
Hmmmm, it looks like this is a non-standard extension right? The only standard I see is  http://spec.graphql.org/draft/#sec-Subscription

Reply:
Yeah, it make senses it is specific to the implementation of subscriptions over WS.